textBasedAdventure.js

```diff
@@ -1,5 +1,157 @@
 const readline = require('readline-sync')
 
-let nameInput = readline.question("Enter your name: ")
+let nameInput = readline.question("Yurrrr washpoppingg? What's your name?: ")
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+console.log(`${nameInput}? Pull up then.`)
+
+let answerInput = readline.question("You ready for this?")
+
+
+if (answerInput === "yes" || "Yes" || "YES") {
+console.log(`${answerInput} ? Good! Start it up!`);
+} else { 
+    console.log("Oh well, we starting it up anyway...");
+} 
+
+
+let ageInput = readline.question("...but wait.. how old are you? 👀")
+
+for (i = 0; i > 17; i -- ) {
+    if (ageInput <= 17) {
+        console.log(ageInput+ "? Damn kid. Access denied.")
+    } else {
+        console.log(ageInput + "? Oh bet, let's get to it.")
+    }
+}
+
+let secretInput = readline.question("I need to make sure you're not a robot. What's 9 + 10?")
+
+let storyNodeOne = readline.question("You are walking to the mall. It is midnight which means it is super dark out and you notice one of those weird vans following behind you real slow. What do you do?   [A] Keep walking! It does not scare me.  [B] I might do a little power walk.  [C] RUNNNN! ... and probably duck down the alley too.")
+
+
+var i = 0;
+while (i > 20 && i < 22) {
+  console.log(storyNodeOne);
+  i++
+}
+
+let storyNodeTwo = readline.question("You see 2 doors. Door A is cracked open a bit and Door B is shut real tight. What do you do.  [A] Bang on door B! I'm not entering some random door or    [B] I'm going through Door A")
+
+if (storyNodeOne === "a") {
+    console.log("Congratulations. You've been kidnapped. Game OVER!");
+} else if (storyNodeOne === "b") {
+        console.log("I mean... I see your point but you still got kidnapped :( Game over.");
+} else if (storyNodeOne === "c") {
+        console.log("Oh, you smart. You're safe... for now." + storyNodeTwo);
+} else {
+        console.log("Error. Start game over.");
```

Comment:
You should stop and restart the game, instead your game keeps going and execute the next line of code